app/views/transactions/_transfer_form.html.slim

```diff
@@ -0,0 +1,16 @@
+= simple_form_for(transaction, url: transactions_transfer_path, remote: true, html: {id: 'new_transfer_form'} ) do |f|
```

Comment:
@Gataniel this form should be based on Transfer model, not on Transaction. And Transfer model creates required transaction objects.